crypto/evp/evp_lib.c

```diff
@@ -709,6 +709,10 @@ int EVP_CIPHER_get_mode(const EVP_CIPHER *cipher)
 
 int EVP_MD_is_a(const EVP_MD *md, const char *name)
 {
+    if (md == NULL) {
+        ERR_raise(ERR_LIB_EVP, EVP_R_MESSAGE_DIGEST_IS_NULL);
```

Comment:
This ERR_raise is debatable. It would be actually a good fallback to just return 0 for NULL md as NULL md is not anything with a name. So this is not really an error.

Reply:
+1, return 0 seems like the way to go.